src/kms/kms.ts

```diff
@@ -129,6 +130,36 @@ export class KmsClient {
     return response.object
   }
 
+  /**
+   * Retrieve a list of KMIP Object from the KMS
+   * @param {string[]} tags list of tags
+   * @returns {KmsObject[]} list of KMIP Objects
+   */
+  public async getObjectsByTags(tags: string[]): Promise<KmsObject[]> {
+    const uniqueIdentifiers = await this.getUniqueIdentifiersByTags(tags)
+    return await Promise.all(
+      uniqueIdentifiers.map(async (uniqueId) => await this.getObject(uniqueId)),
+    )
+  }
+
+  /**
+   * Retrieve a list of unique identifiers from the KMS
+   * @param {string[]} tags list of tags
+   * @returns {string[]} list of unique identifiers in the KMS
+   */
+  public async getUniqueIdentifiersByTags(tags: string[]): Promise<string[]> {
+    const attributes = new Attributes("SymmetricKey")
+    const enc = new TextEncoder()
+    const vendor = new VendorAttributes(
```

Comment:
shouldn't we parametrize those  `cosmian` and `tag` fields? Make them default to `cosmian` and `tag`